test/test_twodnavierstokes.jl

```diff
@@ -239,31 +246,33 @@ function test_twodnavierstokes_advection(dt, stepper, dev::Device=CPU(); n=128,
   Ffh = rfft(Ff)
 
   # Forcing
-  function calcF!(Fh, sol, t, cl, v, p, g)
+  function calcF!(Fh, sol, t, clock, vars, params, grid)
     Fh .= Ffh
+    
     return nothing
   end
 
   prob = TwoDNavierStokes.Problem(dev; nx=n, Lx=L, ν=ν, nν=nν, μ=μ, nμ=nμ, dt=dt, stepper=stepper, calcF=calcF!, stochastic=false)
-  sol, cl, p, v, g = prob.sol, prob.clock, prob.params, prob.vars, prob.grid
-  TwoDNavierStokes.set_zeta!(prob, zetaf)
+  
+  TwoDNavierStokes.set_ζ!(prob, ζf)
 
   stepforward!(prob, nt)
+  
   TwoDNavierStokes.updatevars!(prob)
 
-  isapprox(prob.vars.zeta, zetaf, rtol=rtol_twodnavierstokes)
+  isapprox(prob.vars.ζ, ζf, rtol=rtol_twodnavierstokes)
 end
 
 function test_twodnavierstokes_energyenstrophy(dev::Device=CPU())
   nx, Lx  = 128, 2π
-  ny, Ly  = 128, 3π
+  ny, Ly  = 126, 3π
```

Comment:
I had not noticed there was a test using an anisotropic domain and grid spacing, particularly since all of the examples use nx=ny and dx=dy. Are anisotropic grids supported across all the modules?

Reply:
Also, your compression of the dissipation function looks good to me!

Reply:
> I had not noticed there was a test using an anisotropic domain and grid spacing, particularly since all of the examples use nx=ny and dx=dy. Are anisotropic grids supported across all the modules?

Yes!